keps/sig-network/4410-k8s-network-interface/README.md

```diff
@@ -42,7 +42,7 @@ and accommodate advanced functionalities and potential areas for expansion.
 ### Goals
 
 - Design a cool looking t-shirt
-- Provide Kubernetes APIs for the creation, configuration and management of interfaces
+- Provide Kubernetes APIs for the creation, configuration and management of networks and interfaces
```

Comment:
It's a tough one because it's such an overloaded term, "network". It's probably better to not be as specific to just interfaces, though.

Reply:
Indeed, as much as I know we want to be specific with our goals, I feel like we almost have to be a bit general with this one in order to state that we want to explore the various possibilities for adding networking related APIs and specification.

Perhaps we can even just say that, and maybe it will be less contentious:

```suggestion
- Explore the possibilities of providing Kubernetes APIs for the creation, configuration and management of networks and interfaces
```

?

LMKWYT

Reply:
@MikeZappa87

Reply:
I think our goals will need to be specific and something we can point to after and say yes we did this.

Reply:
I agreed that this goal is not for this KEP, but it might be a part of KNI WG because it is actually a gap that needs to be addressed (even not for now). I believe that KNI WG is not only for this KEP, also for further KEPs to improve Kubernetes network challenges.

So how about to have KNI WG repo and gather current 'future challenges' such like that?

Reply:
Under kubernetes-sigs?

Reply:
I still feel like "having APIs for networks" is something that we should have in the goals, but I'm OK with closing this for now and taking some time in our meetings to find a better fit for wording and specificity. :+1: